package.json

```diff
@@ -29,7 +29,7 @@
     "handlebars": "^4.0.5",
     "mkdirp": "^0.5.0",
     "q": "^1.1.2",
-    "svg2ttf": "^4.0.0",
+    "svg2ttf": "https://github.com/uchihamalolan/svg2ttf.git",
```

Comment:
We need to move `svg2ttf` also under freshdesk, it is not recommended to link personal repos.